service/version.go

```diff
@@ -153,24 +153,30 @@ func (uis *UIServer) versionPage(w http.ResponseWriter, r *http.Request) {
 					DisplayName: t.DisplayName,
 				}}
 
-			// TODO: this loop would probably work better
```

Comment:
This changes page loads for versions that don't have many started tasks to make many more db queries. Could this be reduced to a single aggregation per version, or an aggregation per build?

Reply:
now makes a single indexed find query for the version.